pages/wns.js

```diff
@@ -91,17 +90,16 @@ const Page = () => {
   const registry = new Registry(config.services.wns.endpoint);
 
   const handleRefresh = () => {
-    superagent
-      .post(config.services.wns.endpoint, { query: '{ getStatus { version } }' })
-      .then(({ body: { data: { getStatus: result } } }) => {
+    registry.getStatus()
+      .then(result => {
         setStatus({ ts: Date.now(), result });
 
         registry.queryRecords({ type })
           .then(records => setRecords(records))
           .catch(({ errors }) => setStatus({ ts: Date.now(), result, error: joinErrors(errors) }));
       })
       .catch(error => {
-        setStatus({ ts: Date.now(), error: String(error) });
+        setStatus({ ts: Date.now(), error: error.statusText || 'Connection error.' });
```

Comment:
This looks ugly, but we get a XMLHTTPRequest object in case of an error (e.g. server isn't started).